flink-connector-activemq/pom.xml

```diff
@@ -35,7 +35,7 @@ under the License.
 
     <!-- Allow users to pass custom connector versions -->
     <properties>
-        <activemq.version>5.15.10</activemq.version>
+        <activemq.version>5.16.2</activemq.version>
```

Comment:
is this mandatory for flink update?
if not will be better in other commit, only with this update (and related)